saleor/payment/gateways/braintree/errors.py

```diff
@@ -122,5 +122,4 @@ class BraintreeException(Exception):
     def __init__(self, *args, **kwargs):
         if args or kwargs:
             super().__init__(*args, **kwargs)
-        else:
-            super().__init__(DEFAULT_ERROR_MESSAGE)
+        super().__init__(DEFAULT_ERROR_MESSAGE)
```

Comment:
Codeclimate is raising an issue that there's explicit return in the __init__ now. Should we revert to if..else or ignore it?

Reply:
Hmm, ususally ```__init__``` should return ```None```, so explicit return is not recommended. In this case it should be fine. But maybe let's revert to if-else to remove this warning since there is no warning with ```if-else```. :p